components/datetime/tests/fixtures/tests/components_with_zones.json

```diff
@@ -124,7 +124,7 @@
                     "hour": "two-digit",
                     "minute": "two-digit",
                     "second": "two-digit",
-                    "time_zone_name": "offset"
+                    "time_zone_name": "long-offset"
                 },
                 "semantic": {
```

Comment:
In these cases, I suggest updating the test expectation instead of the inputs.

Reply:
I had to split the offset field into `OffsetLong` and `OffsetShort`, so I had to change this